### DISCUSSION TITLE
add aplayer

### DISCUSSION THREAD

**Discussion by devil6117** [CaiJimmy/hugo-theme-stack#988] [Ideas]
aplayer is a good web-player,could you add aplayer to stack theme?
![image](https://github.com/CaiJimmy/hugo-theme-stack/assets/30503282/989ff99a-51d6-4992-8b04-6ac56ffbb8db)
I try to add it to my blog, but it can't show lrc and other bug...

**Comment by CaiJimmy** [comment]
Sorry, I don't use this player and have no plans to support it due to lack of resources and time 🙏 .

**Comment by devil6117** [comment]
> Sorry, I don't use this player and have no plans to support it due to lack of resources and time 🙏 .

Don't be apologetic, it's not your obligation, I want it when you have time and resources, I enjoy your work. thanks for your theme, it's the best theme for blog on hugo.
